engine/src/main/java/com/datatorrent/stram/stream/BufferServerSubscriber.java

```diff
@@ -242,6 +243,21 @@ public long getByteCount(boolean reset)
     }
 
     @Override
+    public int size(final boolean dataTupleAware)
+    {
+      int size = size();
+      if (dataTupleAware) {
```

Comment:
@tweise : This is used during dynamic partitioning based on backlog. User needs to set the Annotation on the Operator to use this expensive flavor